releasePipeline/30-deploy-maven-galasa.sh

```diff
@@ -72,14 +72,23 @@ mkdir -p temp
 
 export release_type="release"
 
-function ask_user_for_release_version {
-    h1 "Please type the version of this release..."
+function get_galasa_version_to_be_released {
+    h1 "Working out the version of Galasa to test and release."
 
-    read -p "Please enter the version number: " version
+    url="https://development.galasa.dev/main/maven-repo/obr/dev/galasa/dev.galasa.uber.obr/"
+    curl $url > temp/galasa-version.txt -s
+    rc=$?; 
+    if [[ "${rc}" != "0" ]]; then 
+      error "Failed to get galasa version"
+      exit 1
+    fi
 
-    echo "You are doing a release for version ${version}"
-}
+    # Note: We take the 2nd line which has an "<a href" string on... hopefully it won't change...
+    galasa_version=$(cat temp/galasa-version.txt | grep "<a href" | head -2 | tail -1 | cut -f2 -d'"' | cut -f1 -d'/')
```

Comment:
can we look at using grep such as `<a * href=\"[0-9]*\.[0-9]*\.[0-9]*\/\"` instead in order to as that would be a bit more flexible?